### PR TITLE
Validation Updates to support field level validations

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#7145

*Description of changes:*

* Minor updates to the StaffPayment and Certify components

* Rebuilt Storybook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Review by andrepestana-aot — COMMENTED** [review]
Looks good to me. Don't we have any tests for this one?

**Comment by cameron-freshworks** [comment]
> Looks good to me. Don't we have any tests for this one?

I mean, we could add a test for checking the colour of the text but i didn't see a lot of value in it. 
We can see little things like this in Storybook at least :)

**Comment by andrepestana-aot** [comment]
> > Looks good to me. Don't we have any tests for this one?
> 
> I mean, we could add a test for checking the colour of the text but i didn't see a lot of value in it.
> We can see little things like this in Storybook at least :)

I was referring to test the calls to validate the form in StaffPayment.

**Comment by cameron-freshworks** [comment]
> > > Looks good to me. Don't we have any tests for this one?
> > 
> > 
> > I mean, we could add a test for checking the colour of the text but i didn't see a lot of value in it.
> > We can see little things like this in Storybook at least :)
> 
> I was referring to test the calls to validate the form in StaffPayment.

Sure, will add 👍

File: src/components/Certify/Certify.vue [hunk 1]
```diff
@@ -90,10 +91,17 @@ export default class Certify extends Vue {
   @Prop({ default: 10 })
   private secondColumn: number
 
+  /** Call field validations. */
+  @Prop({ default: false })
+  private validate: boolean
+
   /** Promp Error. */
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.vue:98]: Promp?
hehe

**Review by severinbeauvais — APPROVED** [review]
LGTM. Merge when ready.

File: src/components/Certify/Certify.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue:98]: fixed :)

**Comment by cameron-freshworks** [comment]
> LGTM. Merge when ready.

thanks! Just updating some unit tests 👍

File: src/components/Certify/Certify.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Certify/Certify.vue:98]: You got to that promly, haha.

Comment (severinbeauvais) [reply src/components/Certify/Certify.vue:98]: > You got to that "promply", haha.

Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue:98]: 🤣

**Review by andrepestana-aot — APPROVED** [review]
Looks great!

**Pull request merged**